index.html

```diff
@@ -1,145 +1,162 @@
-                    </p>
-                </li>
-                <li>
-                    <h3><a
-                        href='examples/CameraPositionerWidget/example.html'>
-                        Camera Positioner Widget</a></h3>
-                    <p>
-                        The camera positioner is a tool that show the
-                        coordinates and the orientation of the camera.
-                    </p>
-                </li>
-                <li>
-                    <h3><a
-                        href='examples/3DTilesDebugWidget/example.html'>
-                        3DTiles Debug Widget</a></h3>
-                    <p>
-                        3DTiles Debug displays a window which shows various
-                        information about 3DTiles objects and city objects.
-                    </p>
-                </li>
-                <li>
-                    <h3><a
-                        href='examples/CityObjectWidget/example.html'>
-                        City Objects Widget</a></h3>
-                    <p>
-                        This extensions allows the user to explore city objects
-                        and color them according to filters.
-                    </p>
-                </li>                
-                <li>
-                    <h3><a
-                        href='./examples/LayerChoiceWidget/example.html'>
-                        LayerChoice Widget</a></h3>
-                    <p>
-                        This extensions interact with layers
-                    </p>
-                </li>
-                <li>
-                    <h3><a
-                        href='./examples/TemporalWidget/example.html'>
-                        Temporal Widget</a></h3>
-                    <p>
-                        This extensions allows the user to explore city objects
-                        and color them according to filters.
-                    </p>
-                </li>
+              <li>
+                <a href="https://projet.liris.cnrs.fr/vcity/">website</a>
+              </li>
             </ul>
-        </section>
-        <footer>
-            <img id="logoIMU" src="examples/assets/img/logo-imu.png"></img>
-            <img id="logoLIRIS" src="examples/assets/img/logo-liris.png"></img>
-        </footer>
-    </body>
-</html>
\ No newline at end of file
+
+            <h2>Code</h2>
+            <ul>
+              <li><a href="https://github.com/VCityTeam/UD-Viz">github</a></li>
+              <li>
+                <a href="https://www.npmjs.com/package/ud-viz">npm package</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div id="viewer">
+        <div id="projects">
+          <a
+            href="./examples/AllWidget/example.html"
+            target="_blank"
+            rel="noopener"
+            title="This demo implements all widgets and extensions currently available. As of today, this includes :
```

Comment:
Some example descriptions use the term "demo" when others use the term "example".
Use the term "example" everywhere.